pkg/fanal/types/const.go

```diff
@@ -84,3 +84,13 @@ const (
 
 	MixLock = "mix.lock"
 )
+
+// ApplicationTypes contains types  we cat think of as Application
+// Required to `primary purpose` field of SBOM
+// https://github.com/interlynk-io/sbomqs/blob/main/Features.md#primary-purpose
+var ApplicationTypes = []string{
+	GoBinary,
+	RustBinary,
+	Jar,
+	NodePkg,
+}
```

Comment:
I am not sure about this way.
I had idea: use struct instead of string for `Type` field. 
Something like this:
```
type Type struct {
	name  string
	isApp bool // true for `jar`, `goBinary`, etc...
}
```